data/help/texts.json

```diff
@@ -5,8 +5,8 @@
     "name": "<a|A>: Introduction",
     "messages": [
       "Cataclysm: Dark Days Ahead is a turn-based survival game set in a post-apocalyptic world.  You have survived the original onslaught, but the future looks pretty grim.",
-      "You must prepare to face the many hardships to come including dwindling supplies, hostile creatures, and harmful weather.  Even among fellow survivors you must stay alert, since someone may be plotting behind your back to take your hard-earned loot.",
-      "Though one can think Cataclysm: Dark Days Ahead is a roguelike, it vastly differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival game, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on the principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
+      "You must prepare to face the many hardships to come, including dwindling supplies, hostile creatures, and harmful weather.  Even among fellow survivors you must stay alert, since someone may be plotting behind your back to take your hard-earned loot.",
+      "Though one might think Cataclysm: Dark Days Ahead is a roguelike, it vastly differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival game, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on the principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
```

Comment:
Hmm. "In other respects" implies to me that it's more than, or differs significantly from, your typical roguelike to such an extent that thinking of Cataclysm as simply "a roguelike" will lead a player astray. My usage of "might" here attempts to capture that difference.

Reply:
Perhaps:
```suggestion
      "Though one might think of Cataclysm: Dark Days Ahead as a roguelike, it vastly differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival game, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on the principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
```